planter/planter.sh

```diff
@@ -19,8 +19,8 @@
 set -o errexit
 set -o nounset
 IMAGE_NAME="gcr.io/k8s-testimages/planter"
-VERSION="0.5.4-1"
-IMAGE="${IMAGE_NAME}:${VERSION}"
+TAG="${TAG:-'0.5.4-1'}"
```

Comment:
I don't think you need the single quotes

Reply:
fixed